### PR TITLE
Add info to README

### PR DESCRIPTION
**Pull request by bernars-usa**
Just learned about template repositories (https://help.github.com/en/articles/creating-a-template-repository), which is very cool. Adding information about that here so we can decide what to seed this with.

### CODE REVIEW

**Review by sharmaneil — APPROVED** [review]
looks good!

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# hello-world
\ No newline at end of file
+# Hello World
```
Comment (sharmaneil) [inline README.md:1]: not Hello Universe?

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
...
+
+Create your own place on GitHub to experiment. This repository is a [template repository](https://help.github.com/en/articles/creating-a-repository-from-a-template), so copy away!
```
Comment (sharmaneil) [inline README.md:3]: test

**Comment by sharmaneil** [comment]
@bernars-usa can you take al ook at this?

**Comment by bernars-usa** [comment]
Eh, let's keep this as Hello World. People know what that phrase means.

**Pull request merged**